packages/invoices/src/InboundShipment/DetailView/GeneralTab.tsx

```diff
@@ -35,48 +43,188 @@ const Expand: FC<{ rowData: InboundShipmentItem }> = ({ rowData }) => {
+            }
+          },
+        },
+      ],
+      [
+        'expiryDate',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'expiryDate', '');
+            } else {
+              return rowData.expiryDate;
+            }
+          },
+        },
+      ],
+      [
+        'locationName',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'locationName', '');
+            } else {
+              return rowData?.locationName;
+            }
+          },
+        },
+      ],
+      [
+        'sellPricePerPack',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'sellPricePerPack', '');
+            } else {
+              return rowData.sellPricePerPack;
+            }
+          },
+        },
+      ],
+      [
+        'packSize',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return ifTheSameElseDefault(lines, 'packSize', '');
+            } else {
+              return rowData.packSize;
+            }
+          },
+        },
+      ],
+      [
+        'unitQuantity',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return lines.reduce(getUnitQuantity, 0);
+            } else {
+              return rowData.packSize * rowData.numberOfPacks;
+            }
+          },
+        },
+      ],
+      [
+        'numberOfPacks',
+        {
+          accessor: ({ rowData }) => {
+            if ('lines' in rowData) {
+              const { lines } = rowData;
+              return lines.reduce(getSumOfKeyReducer('numberOfPacks'), 0);
+            } else {
+              return rowData.numberOfPacks;
+            }
+          },
+        },
+      ],
+      getRowExpandColumn(),
       GenericColumnKey.Selection,
     ],
-    { onChangeSortBy: onSort, sortBy },
-    [sortBy]
+    {},
+    []
   );
 
   return (
-    <DataTable
-      onRowClick={onRowClick}
-      ExpandContent={Expand}
-      pagination={{ ...pagination, total: activeRows.length }}
-      columns={columns}
-      data={activeRows.slice(
-        pagination.offset,
-        pagination.offset + pagination.first
-      )}
-      onChangePage={pagination.onChangePage}
-      noDataMessage={t('error.no-items')}
-    />
+    <Grid container flexDirection="column" flexWrap="nowrap" width="auto">
+      <Grid
+        item
+        justifyContent="flex-start"
+        display="flex"
+        flex={0}
+        sx={{ padding: '5px', paddingLeft: '15px' }}
+      >
+        <Switch
```

Comment:
I now wonder if this should also be sticky at the top.. but don't even want to think about it right now

Reply:
I was somewhat wondering if this could live in the 'filter' popover that is in the requisition examples. Maybe for invoices we have one with a toggle for grouping and .. uhh.. something else

Reply:
that's a good idea - nice use of the popover filter thingy